weave-js/scripts/generate-schemas.sh

```diff
@@ -3,13 +3,16 @@
 # Exit on error
 set -e
 
-SCHEMA_INPUT_PATH="../weave/trace_server/interface/builtin_object_classes/generated/generated_builtin_object_class_schemas.json"
-SCHEMA_OUTPUT_PATH="./src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/generatedBuiltinObjectClasses.zod.ts"
+SCHEMA_PATH="../weave/trace_server/interface/builtin_object_classes/generated/generated_builtin_object_class_schemas.json"
+OUTPUT_PATH="./src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/generatedBuiltinObjectClasses.zod.ts"
 
 echo "Generating schemas..."
 
+# First resolve circular references in the schema
+python3 ./resolve_schema_circular_refs.py "$SCHEMA_PATH"
```

Comment:
This doesn't work when run from `make synchronize-base-object-schemas`  because the working directory will be `weave-public/weave-js`
```suggestion
python3 "$(dirname "$0")/resolve_schema_circular_refs.py" "$SCHEMA_PATH"
```

Reply:
Fixed! I thanks - I changed that right before pushing.